presto-main/src/main/java/com/facebook/presto/tdigest/docs/t-digest.md

```diff
@@ -21,6 +21,7 @@ The t-digest algorithm uses a set of centroids to store a distribution of values
 * compression: the compression factor used when creating the t-digest
 * total weight: the total number of values in the t-digest
 * activeCentroids: the number of centroids used by the t-digest
+* currentSize: the current size of the weight/mean arrays which represent the centroids
```

Comment:
Could we use `activeCentroids` and not introduce `currentSize`.  I.e. isn' the active centroid count an effective proxy for the current size?